topics/unity/index.md

```diff
@@ -0,0 +1,11 @@
+---
+aliases: unity3d
+created_by: Unity Technologies
+display_name: Unity
+github_url: https://github.com/Unity-Technologies
+logo: unity.png
+short_description: Unity is a game development platform used to build high-quality 3D, and 2D games, deploy them across mobile, desktop, VR/AR, consoles, or the Web.
+topic: unity
+url: https://unity3d.com/
+---
+Unity is a game development platform used to build high-quality 3D, and 2D games, deploy them across mobile, desktop, VR/AR, consoles, or the Web.
```

Comment:
I think the comma before "and 2D" is unnecessary.

It feels a bit odd to capitalize "the Web".

What do you think about spelling out VR and AR?

Reply:
Made the suggested changes - thanks! ✓ 

> What do you think about spelling out VR and AR?

I think VR/AR is generally well understood... and I prefer the short version here. That said, it we have any hard and fast rules or general preferences I can add it in parenthesis or use some `<acronym>` tags. Let me know 🤘